### PR TITLE
Update docs in preparation for release

### PR DESCRIPTION
**Pull request by wil93**

### CODE REVIEW

File: docs/Installation.rst [hunk 1]
```diff
@@ -136,7 +137,12 @@ Installing CMS and its Python dependencies
 
 There are a number of ways to install CMS and its Python dependencies:
 
-Method 1: Global installation with pip
+Method 1: Using the Docker image
```
Comment (veluca93) [inline docs/Installation.rst:140]: I would not recommend the docker image as the first method.

I'd recommend putting virtual environment first, global installation second, and docker image third.

Once the docker image is a bit more complete (IIRC @Virv12 had some issues with it), we can move it further up.

Comment (wil93) [reply docs/Installation.rst]: As discussed offline, now we recommend venv method, followed by docker method

**Review by veluca93 — APPROVED** [review]

File: docs/Installation.rst (hunk 1 above)
Comment (Virv12) [reply docs/Installation.rst]: Well...

- In the container there are multiple config files
- The database password is hardcoded
- You cannot change the config without rebuilding the image
- There is no entrypoint
- We are installing the dev-dependencies, but I don't know python enough to judge this.

**Review by veluca93 — APPROVED** [review]

File: docs/Installation.rst (hunk 1 above)
Comment (wil93) [reply docs/Installation.rst]: Thanks @Virv12 indeed our Dockerfile seems to be currently focused on the local test / development use case, we should expand it in the future to better support passing custom configuration etc.

**Pull request merged**